test/tracedump.ml

```diff
@@ -22,6 +23,7 @@ let print_error = function
   | `Ambiguous_uri -> prerr_string "More than one provider for a given URI"
 
 let main uri =
+  let _ = Plugins.load() in
```

Comment:
Also, are you sure, that you need to load plugins at all. It looks like that this example program is specific to this particular plugin, and should work out of source code. So there is no need to install and load plugins. I would suggest you to revert to the explicit loading.